docs/DeveloperGuide.md

```diff
@@ -270,24 +272,24 @@ The following sequence diagram shows how the undo operation works:
 The `redo` command does the opposite — it calls `Model#redoAddressBook()`, which shifts the `currentStatePointer` once
 to the right, pointing to the previously undone state, and restores the address book to that state.
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** If the `currentStatePointer` is at index `addressBookStateList.size() - 1`, pointing to the latest address book state, then there are no undone AddressBook states to restore. The `redo` command uses `Model#canRedoAddressBook()` to check if this is the case. If so, it will return an error to the user rather than attempting to perform the redo.
+<div markdown="span" class="alert alert-info">:information_source: **Note:** If the `currentStatePointer` is at index `addressBookStateList.size() - 1`, pointing to the latest address book state, then there are no undone AddressBook states to restore. The `redo` command uses `Model#canRedoAddressBook()` to check if this is the case. If so, it will return an error to the CS2100 TA rather than attempting to perform the redo.
```

Comment:
Should we change the method names to refer to ProgrammerError?

Reply:
I think we can delete this whole chunk since we don't have an undo / redo operation?

Reply:
it will be a fix for #469

Reply:
It is a proposed enhancement, which is allowed. We should discuss this with the rest whether to delete this or replace with a different proposed enhancement.

Reply:
alright, I will unlink the fix first. other than that should be fine?

Reply:
Yup, thank you! 🪂